src/index.html

```diff
@@ -2,11 +2,49 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet"
+          href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--month-length--29">
+        <div
+             class="calendar__day calendar__day--1 calendar__day--start-day--sun">
+        </div>
+        <div class="calendar__day calendar__day--2"></div>
```

Comment:
You don't need to create separate class for each calendar day. Use `nth-child` instead of it